src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -47,11 +49,15 @@ public CustomNodeManager(NodeFactory nodeFactory, MigrationManager migrationMana
             this.nodeFactory = nodeFactory;
             this.migrationManager = migrationManager;
             this.libraryServices = libraryServices;
+
+            //Todo decide were we want to store this
+            customNodeInfoCache = new JsonCache<CustomNodeInfo> ("C:\\Temp\\CustomNodeInforCache.temp");
```

Comment:
Maybe this needs to be in AppData so that it doesn't share between users on a machine